templates/web/bromley/waste/_more_services_sidebar.html

```diff
@@ -0,0 +1,26 @@
+[% IF any_report_allowed OR any_request_allowed OR show_garden_subscribe %]
+    <h3>More services</h3>
+    <ul>
+      [% IF any_report_allowed %]
+        <li><a href="[% c.uri_for_action('waste/report', [ property.id ]) %]">Report a missed collection</a></li>
+      [% END %]
+      [% IF any_request_allowed %]
+        <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
+      [% END %]
+      [% IF show_garden_subscribe %]
+        <li><a href="[% c.uri_for_action('waste/garden', [ property.id ]) %]">Subscribe to Green Garden Waste collection</a></li>
+      [% END %]
+    </ul>
+[% END %]
+
+[% IF c.user.from_body OR c.user.is_superuser %]
+    <h3>Assisted collection</h3>
```

Comment:
Might be worth only showing this heading if either `assisted_collection` or `domestic_collection` is true, otherwise you get a heading with nothing under it.

Spotted this on the Bromley staging site:

<img width="476" alt="image" src="https://user-images.githubusercontent.com/22996/154959412-1c325067-a462-4079-93df-689d94e90662.png">